site/using.md

```diff
@@ -389,6 +396,22 @@ individually:
 fluxctl policy --controller=default:deployment/helloworld --tag='helloworld=prod-*' --tag='sidecar=prod-*'
 ``` 
 
+Manual releases without explicit mention of the target image will
```

Comment:
Does 'image' here include the tag? For a controller with multiple containers and images, does flux support updating a specific image to the latest version, e.g. with
```
fluxctl release --controller=default:deployment/helloworld --update-image=helloworld
```
? If so, would the tag filter be applied? It should.

Indeed, does flux support something like
```
fluxctl release --update-image=helloworld
```
for releasing the latest version of an image to all controllers that run that image? If so, the tag filter(s) should apply there too.

Reply:
> Indeed, does flux support something like
> 
> fluxctl release --update-image=helloworld
> for releasing the latest version of an image to all controllers that run that image? If so, the tag filter(s) should apply there too.

It doesn't -- firstly, taken literally that would replace the image field with an untagged image, which would simply run whichever image is present on the host; secondly, we check that the image exists in the image DB, and an untagged image won't pass that check.